src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocStyleCheck.java

```diff
@@ -346,9 +349,16 @@ private void checkComment(final DetailAST ast, final TextBlock comment) {
      */
     private void checkFirstSentenceEnding(final DetailAST ast, TextBlock comment) {
         final String commentText = getCommentText(comment.getText());
+        final String[] sentences = commentText.split("\\.(?=\\s|$)");
+        String firstSentence = "";
+
+        if (sentences.length > 0) {
+            firstSentence = sentences[0];
+        }
```

Comment:
This can be the result of one immutable stream:
```suggestion
        final boolean isInLineReturnTag = Arrays.stream(WORD_DELIMITER.split(commentText))
                .findFirst()
                .map(inlineReturnTagPattern::matcher)
                .filter(Matcher::find)
                .isPresent();
```

Reply:
Done.